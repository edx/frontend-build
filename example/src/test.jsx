/* eslint-disable global-require */
import React from 'react';
import renderer, { act } from 'react-test-renderer';
import App from './App';

describe('Basic test', () => {
  it('should render', () => {
    const component = <App />;
    const tree = renderer.create(component);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
