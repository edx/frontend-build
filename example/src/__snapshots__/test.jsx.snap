// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test should render 1`] = `
<div>
  <h1>
    The Apples
  </h1>
  <ul>
    <li>
      macintosh
    </li>
    <li>
      granny smith
    </li>
    <li>
      fuji
    </li>
    <li>
      golden delicious
    </li>
  </ul>
</div>
`;
